Create restore page content alignment.

Sirmorrison opened the pull request:
### Resolves

Issue #200 

### What's new

This PR aligns the create an restore page contents.

### Relevant screenshots or logs
<img width="798" alt="image" src="https://user-images.githubusercontent.com/27733432/87964533-4ba02f80-cab2-11ea-81f1-0e385f1ba930.png">

<img width="798" alt="image" src="https://user-images.githubusercontent.com/27733432/87964592-5c50a580-cab2-11ea-854a-3845a64e6b9c.png">


### Additional context

_Add any other context about the problem here._

ademuanthony commented on terminal/terminal.go:
```diff
@@ -6,24 +6,54 @@ import (
+		form.SetBorder(true).SetTitle("Enter some data").SetTitleAlign(tview.AlignCenter).SetRect(30,10,40,10)
+		return tviewApp.SetRoot(form, false).Run()
+	
+	}else{
```
you can invert the condition to do away with the else
```if walletExists {
    return tviewApp.SetRoot(layout, true).SetFocus(layout).Run()
}

ademuanthony commented on terminal/walletloader.go:
```diff
@@ -0,0 +1,83 @@
+	}
+}
+
+func CreateWallet(ctx context.Context, password string, walletMiddleware app.WalletMiddleware) (err error) {
```
the `ctx` param is not used, we can do away with it

ademuanthony commented on terminal/walletloader.go:
```diff
@@ -0,0 +1,83 @@
+
+func CreateWallet(ctx context.Context, password string, walletMiddleware app.WalletMiddleware) (err error) {
+	// first check if wallet already exists
+	//work in progress
```
what is this for, I think it can go

Sirmorrison replied:
this is a future implementation for creating wallet will create a different pr to complete the implementation is only half implemented

Sirmorrison replied:
ctx will be used for blockchain sync function which will be implemented in future

Sirmorrison committed:
fix issue with back btn

Sirmorrison committed:
align return btn to the left
make create and restore btn occupy full screen width
centralize page title
crentralize proceed and reset btn

Sirmorrison committed:
centralize restore page title and error label
